spacewalk/setup/spacewalk-setup.spec

```diff
@@ -234,6 +234,11 @@ if [ $1 = 2 -a -e /etc/tomcat6/tomcat6.conf ]; then
         echo 'module = manage_in_tftpd'                     >> /etc/cobbler/modules.conf
     fi
 fi
+
+if [ $1 = 2 -a -e /etc/sysconfig/tomcat ]; then
+     sed -ri '/\-\-add\-modules java\.annotation,com\.sun\.xml\.bind/!s/JAVA_OPTS="(.*)"/JAVA_OPTS="\1 --add-modules java.annotation,com.sun.xml.bind --add-exports java.annotation\/javax.annotation.security=ALL-UNNAMED --add-opens java.annotation\/javax.annotation.security=ALL-UNNAMED"/' /etc/sysconfig/tomcat
```

Comment:
As discussed at IRC, I am not sure how this will work when we submit internally.

It is true that we are changing a configuration file from other package, but idea is having this at tomcat for SLE and openSUSE.

On the other hand, maybe we should have this at https://github.com/uyuni-project/uyuni/blob/eff5cf8f94f3dcf36b39f12a76afcb6038186027/spacewalk/setup/bin/spacewalk-setup-tomcat?

Reply:
This setup tool is only called when we do a new installation. The good thing is, that we might not need this as we always install new.

With SUMA 4.0 you have to install new and migrate the data. 

Only for Uyuni we might need it as we may not force them to install a second instance.
But if we get it for Uyuni, we will get it for SUMA 4.0 as well.

So I think this is ok as we might need it for uyuni. Or do I overlook something here?

Reply:
> On the other hand, maybe we should have this at https://github.com/uyuni-project/uyuni/blob/eff5cf8f94f3dcf36b39f12a76afcb6038186027/spacewalk/setup/bin/spacewalk-setup-tomcat?

@juliogonzalez `tomcat.java_opts` is [read by `spacewalk-setup-tomcat`](https://github.com/mateiw/spacewalk/blob/c34316fe5b686249828496226cb42e8594aebc47/spacewalk/setup/bin/spacewalk-setup-tomcat#L23). The `%post` scriptlet is there only for the upgrade case.

Reply:
Then LGTM. We just need to keep in mind that, according to Fridrich, this is not going to be a temporary workaround anymore.

I am approving the PR right now, FMPOV you can merge when you are ready, @mateiw